### PR TITLE
Fix conditional for snowflakes to match one pr day in December

### PR DESCRIPTION
**Pull request by falkecarlsen**

### CODE REVIEW

File: stregsystem/static/stregsystem/snow.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 d = new Date();
 
 if(d.getMonth() === 11){
-    for(let snowflakes=0; snowflakes <= Math.min(d.getDate(), 24); snowflakes++){
+    for(let snowflakes=0; snowflakes < Math.min(d.getDate(), 24); snowflakes++){
```
Comment (SEbbaDK) [inline stregsystem/static/stregsystem/snow.js:4]: I think we should add concurrency to this to speed up processing of snowflakes.

**Review by SEbbaDK — APPROVED** [review]
Nice B)

File: stregsystem/static/stregsystem/snow.js (hunk 1 above)
Comment (falkecarlsen) [reply stregsystem/static/stregsystem/snow.js:4]: Great, write an issue for it B)

Comment (Rawrior) [reply stregsystem/static/stregsystem/snow.js:4]: I agree, this is an essential improvement.

**Pull request merged**